Makefile

```diff
@@ -322,6 +334,119 @@ docker.prebuilt:
 	time (cd security/docker && docker build -t ${HUB}/istio-ca-test:${TAG} -f Dockerfile.istio-ca-test .)
 	time (cd security/docker && docker build -t ${HUB}/node-agent-test:${TAG} -f Dockerfile.node-agent-test .)
 
+##################################################################################
```

Comment:
+1, we should also remove the old scripts. 

However I would really appreciate if we didn't pollute the source tree - instead copy the binaries and docker files to ${OUT}, and build there.

Reply:
Yeah I didn't like this either.  I did try a scheme where I had "docker build" pass an arg to the dockerfile on where to find the istio binaries, but it didn't work (docker treated the absolute path as a relative path, presumably because of how "docker build" tries to copy the specified source directory to an isolated location).  I thought about copying everything to OUT but I was afraid too many people would gripe the overhead/complexity wasn't a good tradeoff to meet a philosophical desire.  I can try to do the copy now, or save it for a separate PR.  I'm inclined to defer to another PR (partly to avoid an even larger PR, but also because such a change is more likely to perturb the way that other people build).